frontend/src/constants/goerli-constants.json

```diff
@@ -0,0 +1,748 @@
+{
```

Comment:
wait don't we already have this?
Specifically in `~/constants/goerli-constants.json`?
Maybe delete those and only keep these?

Reply:
done